### PR TITLE
Rs 2023.03.11

### PR DESCRIPTION
**Pull request by rschlaikjer**
Been playing a lot of dragonquest 11 in English recently and man all this Luminary business is confusing

2023-03-11-201550_2160x1286_scrot

### CODE REVIEW

File: system_strings/sysmes_text.en [hunk 1]
```diff
@@ -5673,13 +5673,7 @@ Ciel returns from her visit to the Church's local branch.
...
 ロアの声に逆らうように、^光の中心を目指す。^それは自殺行為に他ならない。
-}
-[sha:6a6e8ee21013645a702cbf05c995bbb5cf487998]{
--- それがどうした、と^頭の隅の他人を黙らせる。^ここから戻る選択肢はない。
 それがどうした、と^頭の隅の他人を黙らせる。^ここから戻る選択肢はない。
-}
-[sha:43a21951246fb02775084d4a9c12b0aaf88eebf4]{
--- ロアは語る。^勝ち目が存在しないのなら、^何を“勝ち”とするべきかを。
 ロアは語る。^勝ち目が存在しないのなら、^何を“勝ち”とするべきかを。
 }
```
Comment (Petrikow) [inline system_strings/sysmes_text.en:5678]: ??

File: script/Ciel/Day 14/14_01B_02_CIEL14B_2C.txt [hunk 2]
```diff
@@ -1,62 +1,80 @@
...
+"If you were to travel as the crow flies―――
+%{/i}
 }
 [sha:8be63c4e78bc6f5af0a89a58b57f5856f81fc738]{
 -- Page 69, Offset 39576.
 -- 光体まで目測で800メートル。
-the distance to the Luminary is 800 meters.
+%{i}
+Eight hundred meters separate you from the Luminary.
+%{/i}
```
[the remark this answers is not in the document]
Comment (rschlaikjer) [reply script/Ciel/Day 14/14_01B_02_CIEL14B_2C.txt:63]: Yeah I'm honestly team "just write the numbers however"

File: script/Ciel/Day 14/14_01B_02_CIEL14B_2_2.txt [hunk 3]
```diff
@@ -1,17 +1,20 @@
...
-If I proceed any further, then it will turn into a fight against Arcueid.
+If I proceed any further,
+ I'm going to have to fight Arcueid.
 }
 [sha:e82302a9600b4918f8217a3cb53f82a9cc60d960]{
 -- Page 10004, Offset 39628.
 -- 公園での戦いは、ただの感情のぶつかりあいだった。
-The fight at the park was nothing more than a clash of emotions.
+When we fought at the park,
+ we were just butting heads over our emotions.
```
[the remark this answers is not in the document]
Comment (rschlaikjer) [reply script/Ciel/Day 14/14_01B_02_CIEL14B_2_2.txt:17]: Hmm not sure it does it for me. Sounds too measured.

File: system_strings/sysmes_text.en (hunk 1 above)
Comment (rschlaikjer) [reply system_strings/sysmes_text.en:5678]: Hmm must have happened when rebasing on the changed file format

**Pull request merged**